### ISSUE TITLE
Calendar Invites don't show up in the body

### ISSUE THREAD

**Issue by nirmit** [nylas-mail-lives/nylas-mail#132]
The body shows as blank.  It seems to be an issue with the "events" attribute in the message object.  This is specific to ICS invite based emails.

This issue can be split up into the below desired functionality:
- [x] (@nirmit completed) Show ICS as attachment
- [ ] (pending review) Show invite content (date, time, timezone if specified, name, location, description, etc) with no action buttons
- [ ] (Unassigned) Implement RSVP action buttons

**Comment by nirmit** [comment]
As a starting point, we can atleast show the ICS as an attachment.  What do you guys think, @seesemichaelj @dweremeichik ?

**Comment by seesemichaelj** [comment]
hm i want to say i've seen this issue, but sending myself a calendar invite from google calendar didn't work (i was able to receive an `invite.ics` attachment as well as a email of the details of the invite).  are you able to recreate this?

i agree, that at a minimum, the ICS should show as an attachment

a nicer improvement would have a dedicated header at the top of the invite that had parsed info from the ICS attachment

we can separate these as two separate issues if desired

**Comment by nirmit** [comment]
Yes.  The reason the Calendar invite works from Google is because the ICS is an attachment to the HTML invitation content that comes along.  If you use exchange, you will see that the ICS is not correctly parsed.

I agree on showing the invite content inline in a better way, but I am not sure if I would be able to implement invite actions - Accept, Tentative or Decline! :(

**Comment by seesemichaelj** [comment]
well i think there's three different iterations of this:
1. show ICS as attachment
1. show invite content (date, time, timezone if specified, name, location, description, etc) with no action buttons
1. implement RSVP action buttons

I'll add these as checkboxes in your first comment on this Issue, and people can "pick" which ones they want to work on, and they'll get checked when merged

**Comment by nirmit** [comment]
Great!  I can work on the ICS attachment one.

**Comment by seesemichaelj** [comment]
ICS attachment completed in #133

**Comment by nirmit** [comment]
In order to complete pt. 2 from the list above, I would suggest including the meeting content along with the attachments.  Since the meeting header wouldn't have any actionable buttons, the users can still open the `.ics` to accept/decline meetings.  Let me know if it makes sense.

**Comment by seesemichaelj** [comment]
I agree

**Comment by dweremeichik** [comment]
Auto closed by merge. Reopening.
